splunk/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>splunk</artifactId>
+    <version>1.6.5</version>
+    <parent>
+        <artifactId>splunk-sdk-java</artifactId>
+        <groupId>com.splunk</groupId>
+        <version>1.0.0</version>
+    </parent>
+    <properties>
+        <!-- <jvm.arg.https.protocol>-Dhttps.protocols=TLSv1.1,TLSv1.2</jvm.arg.https.protocol> -->
```

Comment:
It seems like we have some substantial work to do on the javadoc- could you add a property here to disable javadoc generation for the moment and add another bug to clean up javadoc generation?

`<maven.javadoc.skip>true</maven.javadoc.skip>`

Reply:
Both changes are implemented and code is checked-in.